arrows/core/uv_unwrap_mesh.cxx

```diff
@@ -66,11 +68,13 @@ class uv_unwrap_mesh::priv
 {
 public:
   /// Constructor
-  priv() : spacing(0.005)
+  priv()
+    : spacing(0.005)
   {
   }
 
-  priv(const priv& other)
+  // Not clear why this copy constructor is non-functional
+  priv( VITAL_UNUSED const priv& other )
```

Comment:
Agree with your comment. If it remains, we should at least set it `= default`

Reply:
Removed copy CTOR